docs/src/PolyhedralGeometry/pg.md

```diff
@@ -0,0 +1,19 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg.md"]
+```
+
+# Introduction
+
+
+General textbooks offering details on theory and algorithms include: 
+- Theobald + Joswig
+- Ziegler
```

Comment:
Note that you can use bibtex references (see `docs/oscar_references.bib`; to cite something, use `[bibkey](@cite)`.